### PR TITLE
Upgrade to elasticsearch 1.4.1

### PR DESCRIPTION
**Pull request by milovanderlinden**
This pull request contains everything to upgrade to 1.4.1. Please feel free to adopt it into master or create a new 1.4.1 branch.

Kind regards,
Milo van der Linden

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -149,7 +149,7 @@
 				<artifactId>maven-surefire-plugin</artifactId>
 				<version>${maven-surefire-plugin.version}</version>
 				<configuration>
-					<skip>true</skip>
```
Comment (ncolomer) [inline pom.xml]: This was intendedly added to separate unit tests and integration tests (see executions below). You can run them using:

``` bash
mvn clean test
mvn clean integration-test
mvn clean test integration-test
mvn clean verify # similar to the previous one
```

**Comment by ncolomer** [comment]
Great work and big thanks @milovanderlinden!
I'll add some pointers to the `README.md`, merge into a `1.4.x-tmp` branch (as you know I'm working on a full revamp) and generate a release.

**Comment by ncolomer** [comment]
By any chance, have you tested it on elasticsearch releases 1.x and more?
If I remember correctly, the client protocol is (retro-)compatible starting from release 1.0.0.

**Comment by milovanderlinden** [comment]
Thanks @ncolomer, I appreciate your effort. When the full revamp comes along, I will step back in. I am really interested in adding some extra mappings for addresses that, in the near future, could offer the same functionality as nominatim, but in elasticsearch.

**Comment by milovanderlinden** [comment]
No, but that shouldn't be so hard. If I can find some time later this week,
I will build and test against 1.x versions by altering the pom, running
tests, altering pom, running tests and so on.

2014-12-07 12:03 GMT+01:00 Nicolas Colomer notifications@github.com:

**Comment by ncolomer** [comment]
Merged onto branch [`1.4.x-tmp`](https://github.com/ncolomer/elasticsearch-osmosis-plugin/tree/1.4.x-tmp) and plugin version `1.4.0` released!
I updated [`README.md`](https://github.com/ncolomer/elasticsearch-osmosis-plugin) to reflect this (+ plugged some CI).

Thanks again @milovanderlinden :)

Edit: just noticed the numerous commits that was added to this branch... was not intended, definitely not!

**Pull request closed without merging**